tests/ssg_test_suite/oscap.py

```diff
@@ -117,7 +117,10 @@ def generate_fixes_remotely(formatting, verbose_path):
     ]
     command_operands = ['/{arf_file}'.format(** formatting)]
     if 'result_id' in formatting:
-        command_options.extend(['--result-id', formatting['result_id']])
+        # sanitize id because when using "(all)" profile the parenthesis
```

Comment:
I propose to mark it somehow, so when a fix in `oscap-ssh` is released, these workarounds can be reverted quickly.

Reply:
still needed. keeping the change in a nicer way.